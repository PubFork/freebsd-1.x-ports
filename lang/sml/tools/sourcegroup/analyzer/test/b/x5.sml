signature B = sig type z structure A:A val b:A.t end

and A = sig type t structure B:B val c:B.z end
