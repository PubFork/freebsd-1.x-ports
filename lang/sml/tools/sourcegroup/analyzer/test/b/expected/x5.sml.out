source sml "b/x5.sml"
  import signature A B 
  export signature A B ;

